src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -266,6 +267,13 @@ public void respawn()
 		alive = true;
 	}
 
+	public void moveInstantly(Location moveLocation) {
```

Comment:
Please follow the existing convention of method names that do these sort of things.
I would suggest `simulatePlayerMove` for this.
Also please follow the `Allman` brackets style and add annotations which indicate nullability where possible.

```suggestion
	public void simulatePlayerMove(@NotNull Location moveLocation)
	{
```

Reply:
Javadocs for this method would be a nice thing to have too.

Reply:
I'm not sure about this name now, because this method actually doesn't simulate anything. I will name it for now "fakeMove", because it's not actually that every tick it moves player to position, it just teleports him to destination instantly. Javadocks will be added.

Reply:
I've added JavaDocs but idk if that's enough, about that "fakeMove" name, maybe there's some better name... I don't want to name it "move" for the same reason why I think it shouldn't prefix with simulate. What do you think?